reference-adapter/pspcl-ifix-adapter/src/main/java/org/egov/repository/EventPostingDetailRepository.java

```diff
@@ -0,0 +1,14 @@
+package org.egov.repository;
+
+import org.egov.entity.EventPostingDetail;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface EventPostingDetailRepository extends JpaRepository<EventPostingDetail, Long> {
```

Comment:
Have you kept JPA or changed to JdbcTemplate?

Reply:
changed to JDBCTemplate...